### PR TITLE
qt: increase console command max length

### LINKED ISSUE #17618: GUI: testmempoolaccept in console cuts the end of the txhex
How to reproduce: Build a transaction with ~190 inputs, get the hex and issue `testmempoolaccept [\"{txhex}\"]` in the GUI console. It'll give back a `TX decode failed (code -22)` with a stripped txhex where the end is cut down.

### PR DESCRIPTION
**Pull request by 10xcryptodev**
fix #17618
Tested the examples https://github.com/bitcoin/bitcoin/issues/17618#issuecomment-559538070 and works

### CODE REVIEW

**Comment by tryphe** [comment]
Concept ACK

File: src/qt/rpcconsole.cpp [hunk 1]
```diff
@@ -468,6 +468,7 @@ RPCConsole::RPCConsole(interfaces::Node& node, const PlatformStyle *_platformSty
 
     // Install event filter for up and down arrow
     ui->lineEdit->installEventFilter(this);
+    ui->lineEdit->setMaxLength(INT_MAX);
```
Comment (tryphe) [inline src/qt/rpcconsole.cpp:471]: Suggest `std::numeric_limits<int32_t>::max()` as it's the standard for size limits.  I'm also not really sure a 2GB buffer makes sense here, but then again, what is a sane amount? Somewhere between 2^20 and 2^31 seems fine, but towards the latter seems a bit overboard.

Comment (10xcryptodev) [reply src/qt/rpcconsole.cpp]: thanks @tryphe changed for the standard size limit. Yes, hard question about what number should be there. 
What rpc command should be the largest one? Maybe importblock with a full block?
16MB should be fine?

Comment (tryphe) [reply src/qt/rpcconsole.cpp]: I think it's hard to answer that. If we make it just big enough for the biggest command we can come up with, someone will come up with a bigger one. I also don't see any way someone could accidentally paste 2GB of data in the textbox, which at worst would just make things a bit slow. I opt to wait for more comments to see if anyone objects.

**Comment by laanwj** [comment]
Concept ACK. I'd prefer using a limit that is actually testable, though. 16MB sounds good.

> which at worst would just make things a bit slow.

It could also crash the application with an out-of-memory error, or running out of virtual memory when compiled for 32-bit.

**Comment by MarcoFalke** [comment]
The largest p2p message type is a 4 MB block? With hex encoding that should be 8 MB. Wrapped again in json, that should be less than 16 MB.

Btw doc is here: https://doc.qt.io/qt-5/qlineedit.html#maxLength-prop

**Comment by 10xcryptodev** [comment]
@laanwj @MarcoFalke thanks! changed to 16MB and first tests ok

**Comment by MarcoFalke** [comment]
Approach ACK fc6a637a013daeb14b2f93652d7f494f3b8462aa

**Review by luke-jr — APPROVED** [review]
utACK

**Review by hebasto — APPROVED** [review]
ACK fc6a637a013daeb14b2f93652d7f494f3b8462aa, I have reviewed the code and it looks OK, I agree it can be merged.

**Pull request merged**